### PR TITLE
cdx-53-hotspotter-visualization

### PR DESCRIPTION
**Pull request by erinz2020**
1. on match result page, add a switch 'Inspect match area' to show hotspotter images of sight/match candidates annotation's heatmap.
2. when there is no hotspotter url or url is not accessible, the switch will be disabled.

### CODE REVIEW

File: src/pages/match/MatchSighting.jsx [hunk 1]
```diff
@@ -214,12 +246,20 @@ export default function MatchSighting() {
             />
           )}
         </div>
+        <div style={{display: "flex", flexDirection: 'row', alignItems: 'flex-end'}}>
+          <Switch 
+                      checked={checked} 
+                      onChange = {() => handleChange()}
+                      disabled={!urlOK || !heatMapUrl}
+          />
+          <span style={{marginBottom: 10, marginRight: 10}}> Inspect match area </span>
```
Comment (Atticus29) [inline src/pages/match/MatchSighting.jsx:255]: Maybe internationalize?

Comment (erinz2020) [reply src/pages/match/MatchSighting.jsx]: Added. Thank you!

**Review by Atticus29 — APPROVED** [review]
Everything else LGTM, but I didn't test out the functionality. Sorry for hopping on; I was just curious haha.

**Review by naknomum — APPROVED** [review]
looks good

**Pull request merged**